docs/source/platform/schema-validation.md

```diff
@@ -80,90 +123,40 @@ type Query {
 
-Since this is an _additive_ change, and doesn't actually change the default behavior of the `getUsers` query, this isn't a breaking change!
+<h2 id="github">Continuous Integration and GitHub</h2>
 
-<h3 id="breaking">Breaking changes</h3>
-
-An example of a breaking change on an argument would be renaming (or deleting) an argument.
-
-```graphql
-type Query {
-  # What we have.
-  getUsers(ids: [ID!], groupId: ID!): [User]!
-
-  # What we want to end up with.
-  getUsers(ids: [ID!], groupIds: [ID!]): [User]!
-}
-```
-
-There's no way to mark an argument as deprecated, but there are a couple options.
-
-If we wanted to leave the old `groupId` argument active, we wouldn't need to do anything; adding a new argument isn't a breaking change as long as existing functionality doesn't change.
-
-Instead of supporting it, if we wanted to remove the old argument, the safest option would be to create a new field and deprecate the current `getUsers` field.
-
-Using an API management tool, like the Apollo platform, it’s possible to determine when usage of an old field has dropped to an acceptable level and remove it.  The previously discussed [field rollover](#field-rollover) section gives more info on how to do that.
-
-Of course, it’s also possible to leave the field in place indefinitely!
-
-<h2 id="cli">Checking schema changes with the Apollo CLI</h2>
-
-To check and see the difference between the current published schema and a new version, run the following command, substituting the appropriate GraphQL endpoint URL and an API key:
-
-> An API key can be obtained from a service's _Settings_ menu within the [Apollo Engine dashboard](https://engine.apollographql.com/).
-
-```bash
-apollo service:check --key="<API_KEY>" --endpoint="http://localhost:4000/graphql"
-```
-
-> For accuracy, it's best to retrieve the schema from a running GraphQL server (with introspection enabled), though the CLI also reference a local file. See [config options](../platform/apollo-config.html) for more information.
-
-After analyzing the changes against current usage metrics, Apollo will identify three categories of changes and report them to the developer on the command line or within a GitHub pull-request:
-
-1. **Failure**: Either the schema is invalid or the changes _will_ break current clients.
-2. **Warning**: There are potential problems that may come from this change, but no clients are immediately impacted.
-3. **Notice**: This change is safe and will not break current clients.
-
-The more [performance metrics](./performance.html) that Apollo has, the better the report of these changes will become.
-
-<h2 id="github">GitHub Integration</h2>
+Schema validation is best used when integrated in a team's development workflow. To make this easy, Apollo integrates with GitHub to provide status checks on pull requests when schema changes are proposed. To enable schema validation in GitHub, follow these steps:
```

Comment:
I was really expecting a list of steps here, but I don't see one? Might be a nitpick, but we should probably have a 1. 2. 3. if we're going to lead with "follow these steps:"